### DISCUSSION TITLE
Display Group Labels or Direct Selects

### DISCUSSION THREAD

**Discussion by axcrowe** [sstaub/eosstreamdeckplus#6] [Ideas]
Loving using this module, but the one thing I would love to see is adding group labels as a variable. 
I'm trying to use this as a simplified interface for the gaffer on a TV show, and I'm trying to add a page of groups, like direct selects. These will change frequently so labeling them in companion would be cumbersome. 

Alternatively, any way to make streamdeck buttons work just like direct selects?

Thanks for making this awesome module, and for considering looking into these additions.

**Comment by sstaub** [comment]
For DS feature make an issue on https://github.com/bitfocus/companion-module-etc-eos
You can create Group buttons (without the labels from console) by sending an OSC custom command.

**Reply by axcrowe** [reply]
Thanks for the quick reply, I'll post there. 
I've got group buttons working, without labels like you said. I'm just very interested in having the labels show up.
